federation/cluster/common.sh

```diff
@@ -45,7 +45,7 @@ if [[ -z "${FEDERATION_PUSH_REPO_BASE}" ]]; then
     fi
 fi
 
-FEDERATION_IMAGE_REPO_BASE=${FEDERATION_IMAGE_REPO_BASE:-'gcr.io/google_containers'}
+FEDERATION_IMAGE_REPO_BASE=${FEDERATION_IMAGE_REPO_BASE:-gcr.io/google_containers}
```

Comment:
It is unrelated, but my text editor goes bananas with the quotes in place.  I'll drop them from this.